tests/test_modeling_bert.py

```diff
@@ -405,6 +409,309 @@ def create_and_check_for_multiple_choice(
         )
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.num_choices))
 
+    def create_and_check_tracing_for_causal_lm(
```

Comment:
This looks like it could be a common test instead of repeating the same thing for all classes. I suggest adding an attributed `test_tracing` in the common tester and a common test for tracing that will loop through `self.all_model_classes` in the `test_modeling_common` file.